docs/api-docs/apps/apps-quick-start.mdx

```diff
@@ -49,10 +49,10 @@ Next, register the draft app in the [Developer Portal](https://devtools.bigcomme
 2. Click **Create an app**.
 3. Give the app a name.
 4. Click **Technical**.
-5. Enter the **Auth Callback URL** as `https://{ngrok_id}.ngrok.io/api/auth{:http}`. For example, `https://12345.ngrok.io/api/auth{:http}`. You can get the `ngrok_id` from the terminal that is running `ngrok http 3000{:shell}`.
-6. Enter the **Load Callback URL** as `https://{ngrok_id}.ngrok.io/api/load{:http}`.
-7. Enter the **Uninstall Callback URL** as `https://{ngrok_id}.ngrok.io/api/uninstall{:http}`.
-8. If your app supports multiple users, locate the **App Features** section. Select **Multiple Users**, then enter the **Remove User Callback URL** as `https://{ngrok_id}.ngrok.io/api/remove_user{:http}`.
+5. Enter the **Auth Callback URL** as `https://{ngrok_url}/api/auth{:http}`. For example, `https://12345.ngrok.io/api/auth{:http}`. You can get the `ngrok_url` from the terminal that is running `ngrok http 3000{:shell}`.
```

Comment:
note: the sample app readme lists this as `ngrok_address` - we might want to use the same language in both locations, whether it's `url` or `address`

Reply:
I prefer URL.